Framework/PythonInterface/plugins/algorithms/LoadVesuvio.py

```diff
@@ -197,12 +231,29 @@ def _raise_error_if_mix_fwd_back(self, spectra):
         Assumes that the spectra is sorted sorted
         """
         if len(spectra) == 1:
+            self._back_scattering = self._is_back_scattering(spectra[0])
             return
         all_back = self._is_back_scattering(spectra[0])
         for spec_no in spectra[1:]:
             if all_back and self._is_fwd_scattering(spec_no):
                 raise RuntimeError("Mixing backward and forward spectra is not permitted."
                                    "Please correct the SpectrumList property.")
+        self._back_scattering = all_back
+
+#----------------------------------------------------------------------------------------
+
+    def _raise_error_if_non_valid_mode(self, mode, back_scattering):
+        """
+        Checks that the input are valid for the Mode of operation selected
+        SingleDifference - Forward Scattering
+        DoubleDifference - Back Scattering
+        """
+        if mode == "SingleDifference":
```

Comment:
Is this true? I thought the back scattering banks could do all 3 modes: single, double and thick? I may be wrong though so could we just double check with Matthew/Giovanni what the valid modes of operation are?